data/faq/nl.yml

```diff
@@ -43,41 +46,80 @@
-        - Alle leerlingen die deelnemen aan de Informatica-olympiade beginnen op hetzelfde moment (niet enkelen om 14u en anderen om 15u).<br>
-        
-        Wij vinden deze overlap met een andere wedstrijd zelf jammer, maar ze was niet te vermijden. Ze is een gevolg van het feit dat we als nationale olympiade rekening moeten houden met de verschillende kalenders van andere wedstrijden in zowel Vlaanderen als in Franstalig België. 
+        In 2017 valt de eerste ronde van de Informatica-olympiade samen met die van de Biologie-olympiade. De biologie-olympiade gaat ook door op school in een computerklas, op vaste tijdstippen. Uw leerlingen mogen voordien of na afloop de Informatica-olympiade afleggen.<br>
+        Wij vinden deze overlap met een andere wedstrijd zelf jammer, maar ze is niet te vermijden. Ze is een gevolg van de langere voorbereidingstijd die we nodig hebben in 2017, en het feit dat we als nationale olympiade rekening moeten houden met de kalenders van andere wedstrijden in zowel Vlaanderen als in Franstalig België. 
+    -
+      question: Is een computerklas noodzakelijk?
+      answer: De eerste ronde is in 2017 gepland als een wedstrijd via de computer. Indien de voorbereidingen daarvoor succesvol zijn, zal de wedstrijd in een computerklas met internetverbinding moeten plaatsvinden.
     -
-      question: Moet er aan de computer gewerkt worden in de eerste ronde ?
-      answer: "Nee, de eerste ronde is enkel een papieren examen."
+      question: Mijn computerklas is niet groot genoeg!
+      answer: |
+        Het zal zeer waarschijnlijk toegelaten worden de eerste ronde in maximaal 3 aparte sessies te organiseren, m.n. 1 sessie per deelnemerscategorie. 
+        We zullen echter wel vragen dat alle deelnemers van eenzelfde categorie en eenzelfde school, de wedstrijd tegelijkertijd afleggen.
+        Ook vragen we in dat geval dat de communicatie tussen deelnemers van verschillende sessies tot een absoluut minimum wordt beperkt, zelfs al behoren ze tot een verschillende categorie.
     -
       question: Moeten mijn leerlingen zich op uw site inschrijven? 
       answer: |
-        Alle deelnemers moeten zich persoonlijk inschrijven op onze site, zodat ze via email hun resultaten toegestuurd krijgen, en als ze in aanmerking komen, ook de uitnodigingen voor de vormingssessies en de finale krijgen.
+        We trachten vanaf 2017 de inschrijvingen te kunnen laten verlopen via het wedstrijdplatform. De details van de inschrijvingsprocedure worden momenteel nog uitgewerkt. 
+        Een geldig emailadres blijft echter wel noodzakelijk, zodat we deelnemers hun resultaten kunnen toesturen, en als ze in aanmerking komen, ook de uitnodigingen voor de vormingssessies en de finale kunnen bezorgen.
```

Comment:
I think the French bebras platform does not include that... actually I think they do not even register the student names before, the students just has to provide their name when participating (the teacher can still check and fix typos during/after the contest)... let's use France experience which is much bigger than our. So let's not put this rule for now.